dataInterface/movies.js

```diff
@@ -30,10 +33,16 @@ module.exports.getAllComments = async (movieId)=>{
   // https://www.mongodb.com/docs/manual/reference/operator/query-comparison/
   // To get only comments made since 1985:
   // const query = {movie_id: ObjectId(movieId), date: { $gt: new Date("January 1, 1985")}}
-  const query = { movie_id: ObjectId(movieId)};
+
+  let movieQuery=module.exports.getByIdOrTitle(movieId)
```

Comment:
This step is unnecessary. The query object on line 38 can just use the movieId that's passed into the argument.